### ISSUE TITLE
what is "vector_drawing"?

### ISSUE THREAD

**Issue by mjrams73** [orlandpm/Math-for-Programmers#7]
when I try to import vector_drawing, I get an error. what am I doing wrong? thanks.

**Comment by SeveneduS** [comment]
@mjrams73 What exactly do you receive in error message? I tried both 02 and 05 Chapters and both work fine for me.

**Comment by mjrams73** [comment]
"No module name 'vector_drawing'". I think I'm figuring it out. total noob. gotta load up vector_drawing.py and then turn it into a notebook, right? I was treating it like a pre-existing package. unfamiliar with modules.

**Comment by mjrams73** [comment]
nope. still no clue how to get it to work. any guidance is greatly appreciated.

**Comment by SeveneduS** [comment]
So I am doing the following steps and everything works fine:
- Install OpenGL by executing the following command "pip install PyOpenGL PyOpenGL_accelerate"  or you can install the standard one by executing appropriate script from the your python installation folder.
- pip install pygame
- Than run the "jupyter-lab" command
- In jupyter navigate (at the left side panel) to your extraction/clone folder of this repository and open required .ipynb lab file.
- Run it step by step from the very begining.

**Comment by mjrams73** [comment]
thanks! I'll let you know how it goes.
